zebra-chain/src/parameters/transaction.rs

```diff
@@ -0,0 +1,8 @@
+//! Transaction consensus and utility parameters.
+
+///The Overwinter version group ID.
+pub const OVERWINTER_VERSION_GROUP_ID: u32 = 0x03C4_8270;
+///The Sapling version group ID.
+pub const SAPLING_VERSION_GROUP_ID: u32 = 0x892F_2085;
+///The Orchard version group ID.
+pub const ORCHARD_VERSION_GROUP_ID: u32 = 0x6A7_270A;
```

Comment:
Two suggestions:
* use the correct version group ID
* name the version group ID after the transaction version, because
  * Sapling transactions can use the old or new version group ID
  * the version group ID is intended to be used in NU5 and later upgrades

The spec says:
> [Orchard proposal] The transaction version number MUST be 4 or 5. If the transaction version number is 4 then the version group ID MUST be 0x892F2085. If the transaction version number is 5 then the version group ID MUST be 0x26A7270A.

Page 106 in https://raw.githubusercontent.com/daira/zips/orchard-circuit/protocol/orchard.pdf

```suggestion
/// The version group ID for version 5 transactions.
///
/// Orchard transactions must use transaction version 5 and this version
/// group ID. Sapling transactions can use v4 or v5 transactions.
pub const TX_V5_VERSION_GROUP_ID: u32 = 0x26A7_270A;
```